openvpn-server/conf/server.conf

```diff
@@ -0,0 +1,170 @@
+port 1194
+
+proto udp
+
+dev tun
+
+ca /etc/openvpn/secrets/ca.crt
+cert /etc/openvpn/secrets/server.crt
+key /etc/openvpn/secrets/server.key  # This file should be kept secret
+
+tls-version-min 1.3 or-highest
+tls-cipher TLS-ECDHE-RSA-WITH-AES-256-GCM-SHA384
+ncp-ciphers AES-256-GCM
+auth SHA256
+dh none
+ecdh-curve secp384r1
+
+remote-cert-tls client
+
+# Network topology
+# Should be subnet (addressing via IP)
+# unless Windows clients v2.0.9 and lower have to
+# be supported (then net30, i.e. a /30 per client)
+# Defaults to net30 (not recommended)
+;topology subnet
```

Comment:
comment above says this should be `subnet`

Reply:
good catch, fixed